tests/test_errors.py

```diff
@@ -0,0 +1,60 @@
+from io import BytesIO
+
+import pytest
+from lxml import etree
+
+from search.errors import NAMESPACE_MAP
+from tests.schema_utils import get_schema_file_path
+
+
+@pytest.mark.parametrize('status_code, name', [
+    (400, 'Bad Request'),
+    (401, 'Unauthorized'),
+    (403, 'Forbidden'),
+    (404, 'Not Found'),
+    (405, 'Method Not Allowed'),
+    (406, 'Not Acceptable'),
+    (408, 'Request Timeout'),
+    (409, 'Conflict'),
+    (410, 'Gone'),
+    (411, 'Length Required'),
+    # Custom response data and headers are removed for HTTP 412.
+    # See Flask issue 3093 for more information.
+    # (412, 'Precondition Failed'),
```

Comment:
@thewilkybarkid @giorgiosironi Flask currently has a dependency of `werkzeug >=0.14`. I can confirm this is fixed in version `0.15.dev0` which hopefully indicates that a release of `0.15` is imminent. Is it worth using `0.15.dev0` now or should we wait for an official release? I'm hesitant to use this in production but since this is not in production, what do you think?

Reply:
A comment capturing this knowledge plus postponing the upgrade is fine for me, since we are not using this code at the moment fixing it is technically unnecessary.